### ISSUE TITLE
YQL code generation failing with unexpected errors

### ISSUE THREAD

**Issue by ibushmarinov** [ytsaurus/ytsaurus#73]
The request 

```
use ytdemo;
pragma yt.DefaultCluster ='ytdemo';


$multiply_member = ($st, $m, $x) -> {
    return $st.$m*$x
};

$struct_multiply = ($struct, $x) -> {
    $members = ["a", "b"];
    $multiplier = EvaluateCode(LambdaCode(($st) -> {
        return FuncCode(
            "AsStruct",
            ListMap(
                $members,
                ($m) -> (
                    ListCode(
                        AtomCode($m),
                        FuncCode(
                            "Apply",
                            QuoteCode($multiply_member),
                            $st,
                            ReprCode($m),
                            ReprCode($x)
                        )
                    )
                )
            )
        )
    }));
    return $multiplier($struct);
};

insert into `//home/ibushmarinov/testmult`
with truncate
select * from (
    select 
    -- /RenameMembers(
        $struct_multiply(
            TableRow() , 2)
    --     ,
    --      ($x) -> ($x || "_2")),
    -- RenameMembers($struct_multiply(TableRow(), 3), ($x) -> ($x || "_3"))
    from (Select 1 as a, 2 as b)
) Flatten columns;
commit;
select * from `//home/ibushmarinov/testmult`;
```

fails with the following cryptic error: 
```
YQL embedded call failed[1]
yt/yql/plugin/plugin.cpp:171: Failed to run: -memory-:<main>: Fatal: Expression evaluation

    -memory-:<main>:11:19: Fatal: At function: EvaluateCode
    	    $multiplier = EvaluateCode(LambdaCode(($st) -> {
	                  ^
        -memory-:<main>: Fatal: yql/providers/yt/lib/config_clusters/config_clusters.cpp:96: TYtGatewayConfig: No default cluster
        
[1]
```

This seems to be a bug on the server side and not in the code itself. What does "default cluster" even mean in this context?

**Comment by Krisha11** [comment]
Hello!

It was fixed in [19c0638759853903d07f928247367f98960f12a8](https://github.com/ytsaurus/ytsaurus/commit/19c0638759853903d07f928247367f98960f12a8). Please wait for the next [yql-package](https://github.com/ytsaurus/ytsaurus/releases) update.
This example will start working after the update.

**Comment by zlobober** [comment]
Fixed as of today.
